### ISSUE TITLE
Support pandas 1.0

### ISSUE THREAD

**Issue by dialvarezs** [MaxHalford/prince#85]
[Pandas 1.0](https://pandas.pydata.org/docs/whatsnew/v1.0.0.html) removed some functionalities previously deprecated, and some features seems to be broken. For example, trying to fit a FAMD gives the following error:
```
~/.pyenv/versions/3.8.2/envs/bioinformatics/lib/python3.8/site-packages/prince/one_hot.py in transform(self, X)
     29 
     30     def transform(self, X):
---> 31         return pd.SparseDataFrame(
     32             data=super().transform(X),
     33             columns=self.column_names_,

TypeError: SparseDataFrame() takes no arguments
```

**Comment by MaxHalford** [comment]
Yeah I got to fix this...

**Comment by CodePanda9** [comment]
Hi, trying to use MCA from your package and I get the same error. Any idea what I need to change to make it work? I really need to make that plot :(

**Comment by dialvarezs** [comment]
@CodePanda9 you could downgrade pandas fow now. How are you installing the packages?

**Comment by UzielLinares** [comment]
@CodePanda9 try to replace occurrences of SparseDataFrame with a DataFrame with SparseArrays, that should work.

**Comment by MaxHalford** [comment]
Ok I've just released version 0.7.0. Unit tests are passing with pandas 1.0 so everything should be fine. I'll let @dialvarezs close this.

**Comment by dialvarezs** [comment]
Is working for me as well, thank you very much @MaxHalford !
